.npmignore

```diff
@@ -0,0 +1,14 @@
+node_modules/*
+es*/compilers/*
+.DS_Store
+.ruby-version
+_site/
+.idea/
+es5/
```

Comment:
github is for development, npm is for consumption. packages published to npm should have everything - that way it's available when consuming previously-installed-things offline as well.